src/esthri/src/main.rs

```diff
@@ -244,17 +241,22 @@ fn is_aws_compatibility_mode() -> bool {
     is_run_as_aws || has_aws_env_var
 }
 
+/// Allows for an escape hatch to be set that triggers always falling back from
+/// esthri into the real aws tool.
+fn should_always_fallback_to_aws() -> bool {
+    env::var("ESTHRI_AWS_ALWAYS_FALLBACK").is_ok()
```

Comment:
Would it be possible to switch this to check that it's equal to `true` or `1` etc? There isn't an easy way to unset a variable from a workflow config, so workflows would have to write python/bash to do it. But if it's a true/false thing you could make it an `env` resource and toggle it. Or maybe we could make it a proper field in the workflow config?

Reply:
If we go with adding it to the config proper than unsetting it would be fine

Reply:
That's a great point, I like the idea of using an env resource for this. I avoided it initially because I thought it'd be hard to know if the value should be `true` or `True` or `1` or `TRUE` but I guess I can probably just lowercase it in the code and check for equality to `true` or `1`. Will make it much simpler to update workflows to turn it on/off which is great.

IMO it makes a bit more sense as a env resource than as a permanent config option as it should hopefully be only a temporary thing that we have while rolling it out. Eventually hopefully all workflows can use it and don't need to turn it off.